src/main.rs

```diff
@@ -12,6 +12,9 @@ struct Opts {
     /// config directory for aardvark, usually path to a tmpfs.
     #[clap(short, long)]
     config: Option<String>,
+    /// Tells if current netavark invocation is for rootless container.
+    #[clap(short, long)]
+    rootless: Option<bool>,
```

Comment:
Does this allow just `--rootless`? It should work similar to the cobra go library, e.g. if you have a bool option you can use `--rootless` and also `--rootless=true|false` but not `--rootless true|false`

Reply:
Yes it should support both.